scripts/build-images

```diff
@@ -58,6 +58,17 @@ xargs -n1 -t docker image pull --quiet << EOF > build/images-vsphere.txt
     ${REGISTRY}/rancher/mirrored-k8scsi-livenessprobe:v2.1.0
 EOF
 
+xargs -n1 -t docker image pull --quiet << EOF >> build/images-multus.txt
```

Comment:
Otherwise we get another set of images appended every time we build locally
```suggestion
xargs -n1 -t docker image pull --quiet << EOF > build/images-multus.txt
```

Reply:
Good point! Thanks

Reply:
Still needs changed